literature.bib

```diff
@@ -2335,7 +2289,7 @@ @article{STGR:TOSEM20
 
 @inproceedings{KKSL:VaMoS20,
 	author = {Kr{\"{u}}ger, Jacob and Krieter, Sebastian and Saake, Gunter and Leich, Thomas},
-	title = {{EXtracting product lines from vAriaNTs (EXPLANT)}},
+	title = {{EXtracting Product Lines From vAriaNTs (EXPLANT)}},
```

Comment:
Maybe this is an exception where `From` should be `from`.

Reply:
yes, well spotted